main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/edgarduran/Go-Dispatch-Bootcamp/controllers"
+	"github.com/edgarduran/Go-Dispatch-Bootcamp/router"
+)
+
+func main() {
+	fmt.Printf("Hello\n")
+	c := controllers.New()
+	router := router.Setup(c)
+
+	fmt.Printf("Listening on port 3030\n")
+	http.ListenAndServe(":3030", router)
```

Comment:
Please avoid the use of magic naming and magic numbers.

Instead of having them hardcoded in your code, you can get these values from a config file and/or environment variables.

Lucky for us we have some useful packages in Go that will help us with that; please give it a look to either [viper](https://github.com/spf13/viper), [envconfig](https://github.com/kelseyhightower/envconfig), or [go-toml](https://github.com/pelletier/go-toml) (there are many other alternatives, but these are the most famous).

The idea is to have these values as part of a config file; let's imagine the scenario in which you need to change the port of your server. In order to change it and make this change go live you will need to actually modify the code, but if you have these values as part of a `config file` you only need to change a value there, and that it, your source code will remain intact.

```suggestion
	// get the variable from some place (not actually the solution, just an idea how to do that)
	serverPort := config.Get("server_port")
	
	// prepare the address
	serverAddress := fmt.Sprintf(":%d", serverPort)
	
	// use the address
	http.ListenAndServe(serverAddress, router)
```